<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QWidget" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1278</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1278</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music</string>
  </property>
  <widget class="QLabel" name="labelSongName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(100,100,100)</string>
   </property>
   <property name="text">
    <string>随听</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDestroy">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoop">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>730</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>39</width>
     <height>39</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_2">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>630</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Time">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>720</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(66,66,66)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNextSong">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>730</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMIN">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>740</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLiking">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>730</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>54</width>
     <height>54</height>
    </size>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(100,100,100)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonVolum">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>730</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>190</y>
     <width>651</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>641</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonHome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLastSong">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUsed">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>700</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(66,66,66)</string>
   </property>
   <property name="text">
    <string>纵享音乐，极乐随听!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLike">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSuspectSong">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>730</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTheme">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
